rasa/core/training/interactive.py

```diff
@@ -817,7 +820,9 @@ def _write_stories_to_file(
                 isinstance(event, UserUttered) for event in tracker.applied_events()
             ):
                 i += 1
-                f.write("\n" + tracker.export_stories(SAVE_IN_E2E))
+                f.write(
+                    "\n" + tracker.export_stories(SAVE_IN_E2E, False, guessed_format)
```

Comment:
Maybe could be better not use positional parameters and pass each parameter like this line:
https://github.com/FGA-GCES/rasa/blob/72579559b717182d4f3190b87bf5322fe02618b3/rasa/core/training/interactive.py#L816

This `False` parameter is not good to the code.